scripts/notifications.coffee

```diff
@@ -0,0 +1,78 @@
+# Description:
+#   Notifications for botgt
+# Commands:
+#   hubot notify <message> - notify everyone with something
```

Comment:
I think this is bad default behavior @jacobzipper @illegalprime. We should force people to specify the medium so they don't accidentally blast a message to everyone.